getting_started.md

```diff
@@ -4,19 +4,50 @@ permalink: /gettingstarted/
+
+# Getting Started - "Limited" Flavor
+
+For the the "Limited" flavor, you will implement:
+
+* Discovery Service Integration - _Integrate the Seamless Access service to use it as your Discovery Service._
+
+See the [Integration Guide](/integration/) to learn how to configure your Service Provider (SP) software stack.
+
+# Getting Started - "Standard" Flavor
+
+For the "Standard" flavor, you will implement:
+
+* Discovery Service Integration - _Integrate the Seamless Access service to use it as your Discovery Service._
+* Display of Seamless Access Login Button - _Use the Seamless Access service to display the login button component on your SP login page._
+* Integration of Login Button with your SAML SP - _Integrate the Seamless Access service to use it as your Discovery Service._
+
+See the [Integration Guide](/integration/) for details on how to implement.
+
+# Getting Started - "Advanced" Flavor
+
+The Advanced option provides significant control the display and behavior of the software and service, though requires you to run the software yourself. Running your own instance of thiss-js involves two things:
 
 1. An MDQ service that supports the [search API extension](https://github.com/IdentityPython/pyFF/wiki/Extensions-to-MDQ)
 2. The [thiss-js](https://github.com/TheIdentitySelector/thiss-js) software
 
-The [pyFF.io](https://pyff.io) implemenation of MDQ is known to work with thiss-js but other MDQ implemenations should also be made to work with some effort. The main issue is that the MDQ service nees to support search on the /entities/ endpoint by passing a query-parameter ('q') containing search terms. Exactly how seardch is implemented is up to the implmementation but the caller (thiss-js) expects application/json response as described in the README of [thiss-ds-js](https://github.com/TheIdentitySelector/thiss-ds-js).
+The [pyFF.io](https://pyff.io) implementation of MDQ is known to work with thiss-js but other MDQ implementations should also be made to work with some effort. The main issue is that the MDQ service needs to support search on the /entities/ endpoint by passing a query-parameter ('q') containing search terms. Exactly how search is implemented is up to the implementation but the caller (thiss-js) expects application/json response as described in the README of [thiss-ds-js](https://github.com/TheIdentitySelector/thiss-ds-js).
 
 Running an instance of thiss-js can be done by launching a docker container. A simple setup assuming a pyFF backend is described in the README of [thiss-js](https://github.com/TheIdentitySelector/thiss-ds-js).
+
+While some details for implementation of the "Advanced" flavor can be found in the[Integration Guide](/integration/), more detailed documentation can be found in the [Discovery Service Client API Repository README](https://github.com/TheIdentitySelector/thiss-ds-js/blob/master/README.md).
+
+# An Example Implementation
```

Comment:
Since I'm updating the thiss.io site, this clearly won't point to the most recent version of the demo. I'm guessing this documentation will move over to the new domain as well?